pkg/apis/core/validation/validation.go

```diff
@@ -4249,22 +4249,6 @@ func ValidateService(service *core.Service) field.ErrorList {
 		allErrs = append(allErrs, field.NotSupported(specPath.Child("type"), service.Spec.Type, supportedServiceType.List()))
 	}
 
-	if service.Spec.Type == core.ServiceTypeLoadBalancer {
-		portsPath := specPath.Child("ports")
-		includeProtocols := sets.NewString()
-		for i := range service.Spec.Ports {
-			portPath := portsPath.Index(i)
-			if !supportedPortProtocols.Has(string(service.Spec.Ports[i].Protocol)) {
```

Comment:
It looks like we dropped this check which ensures ports are using valid protocol names, I think we should keep it and only drop the validation below for using more than 1 protocol.

Reply:
@thockin had this comment: https://github.com/kubernetes/kubernetes/pull/94028#discussion_r471735567
So I removed it from here as part of that block.

Reply:
ack, thanks